### PR TITLE
Clean up formats in stores topics folder (part 1)

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This was primarily as a fix for #26 (convert tables in stores), but there are other formatting and style issues to address in these topics. 

- Converted remaining HTML tables to MD
- Replaced H2/H3 infinitives with present tense verb headings (better for machine translation and findability)
- Adjusted some procedures to better use numbered steps, as well as clarify navigation and simplify the instructions
- Added alt text for images where needed
- A few misc. formats to align with MD linting rules
- Adjusted icon/selector images where needed to fix inline display

**_Note:_** To keep the PR from being too large, this is the first of a series to address #26

## Affected documentation pages

https://docs.magento.com/m2/ee/user_guide/stores/admin-account.html
https://docs.magento.com/m2/ee/user_guide/stores/admin-actions-control.html
https://docs.magento.com/m2/ee/user_guide/stores/admin-dashboard.html
https://docs.magento.com/m2/ee/user_guide/stores/admin-email-templates-configure.html
https://docs.magento.com/m2/ee/user_guide/stores/admin-global-search.html
https://docs.magento.com/m2/ee/user_guide/stores/admin-grid-controls.html
https://docs.magento.com/m2/ee/user_guide/stores/admin-grid-layout.html
https://docs.magento.com/m2/ee/user_guide/stores/admin-message-inbox.html
https://docs.magento.com/m2/ee/user_guide/stores/admin-session-lifetime.html
https://docs.magento.com/m2/ee/user_guide/stores/admin-signin.html
https://docs.magento.com/m2/ee/user_guide/stores/admin-workspace.html
https://docs.magento.com/m2/ee/user_guide/stores/admin.html
https://docs.magento.com/m2/ee/user_guide/stores/attribute-date-time-options.html
https://docs.magento.com/m2/ee/user_guide/stores/attribute-delete.html
https://docs.magento.com/m2/ee/user_guide/stores/attribute-product-create.html
https://docs.magento.com/m2/ee/user_guide/stores/attribute-sets.html
https://docs.magento.com/m2/ee/user_guide/stores/attributes-customer-address-templates.html
https://docs.magento.com/m2/ee/user_guide/stores/attributes-customer-address.html
https://docs.magento.com/m2/ee/user_guide/stores/attributes-customer.html

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

## Additional information

internal staging build 228

### CODE REVIEW

File: src/stores/attributes-customer-address-templates.md [hunk 1]
```diff
@@ -5,46 +5,36 @@ title: Customer Address Templates
 
 You can modify the template that controls the format of customer billing and shipping addresses that appear on printed invoices, shipments, and refunds, as well as in the addresses book of the customer account. If you need to include additional information, you can create [custom attributes]({{ site.baseurl }}{% link stores/attributes-customer.md %}) that are associated with the customer account and [address]({{ site.baseurl }}{% link stores/attributes-customer-address.md %}), and incorporate them into the template.
 
-## Example 1: Short Format
+## Example 1: short format
 
 For Text One Line Template:
 
-        `{% raw %}{{depend prefix}}{{var prefix}} {{/depend}}{{var firstname}} {{depend middlename}}{{var middlename}} {{/depend}}{{var lastname}}{{depend suffix}} {{var suffix}}{{/depend}}, {{var street}}, {{var city}}, {{var region}} {{var postcode}}, {{var country}}{% endraw %}`
+        {% raw %}{{depend prefix}}{{var prefix}} {{/depend}}{{var firstname}} {{depend middlename}}{{var middlename}} {{/depend}}{{var lastname}}{{depend suffix}} {{var suffix}}{{/depend}}, {{var street}}, {{var city}}, {{var region}} {{var postcode}}, {{var country}}{% endraw %}
```
Comment (dobooth) [inline src/stores/attributes-customer-address-templates.md]: Formatting is off here. We should not be displaying the raw tags. I suspect the template needs to be in a code block and the raw tags outside of that.

File: src/stores/attributes-customer-address-templates.md [hunk 2]
```diff
@@ -5,46 +5,36 @@ title: Customer Address Templates
...
-        `{% raw %}{{if city}}{{var city}},  {{/if}}{{if region}}{{var region}}, {{/if}}{{if postcode}}{{var postcode}}{{/if}}<br/>{% endraw %}` 
-        `{% raw %}{{var country}}<br/>{% endraw %}` 
-        `{% raw %}{{depend telephone}}T: {{var telephone}}{{/depend}}{% endraw %}` 
-        `{% raw %}{{depend fax}}<br/>F: {{var fax}}{{/depend}}{% endraw %}` 
-        `{% raw %}{{depend vat_id}}<br/>VAT: {{var vat_id}}{{/depend}}{% endraw %}` 
+        {% raw %}{{depend prefix}}{{var prefix}} {{/depend}}{{var firstname}} {{depend middlename}}{{var middlename}} {{/depend}}{{var lastname}}{{depend suffix}} {{var suffix}}{{/depend}}{{depend company}}{{var company}}{{/depend}}{{if street1}}{{var street1}}{{/if}}{{depend street2}}{{var street2}}{{/depend}}{{depend street3}}{{var street3}}{{/depend}}{{depend street4}}{{var street4}}{{/depend}}{{if city}}{{var city}},  {{/if}}{{if region}}{{var region}}, {{/if}}{{if postcode}}{{var postcode}}{{/if}}{{var country}}{{depend telephone}}T: {{var telephone}}{{/depend}}{{depend fax}}F: {{var fax}}{{/depend}}{{depend vat_id}}VAT: {{var vat_id}}{{/depend}}{% endraw %}
```
Comment (dobooth) [inline src/stores/attributes-customer-address-templates.md]: Same here.

File: src/stores/attributes-customer-address-templates.md (hunk 1 above)
Comment (leslietilling) [reply src/stores/attributes-customer-address-templates.md]: The `{% raw %}` is a liquid tag and isn't displayed even within the ` enclosure. However, I changed this to use the fenced in block and the raw designation outside of that. The display is exactly the same. The linting rules don't like it, but that happens with some of our other liquid tags too.

File: src/stores/attributes-customer-address-templates.md (hunk 2 above)
Comment (leslietilling) [reply src/stores/attributes-customer-address-templates.md]: same as above

**Comment by leslietilling** [comment]
running tests

**Pull request merged**